apple/Sources/FerrostarMapLibreUI/Views/PortraitNavigationView.swift

```diff
@@ -49,7 +51,10 @@ public struct PortraitNavigationView: View, CustomizableNavigatingInnerGridView
         navigationState: NavigationState?,
         minimumSafeAreaInsets: EdgeInsets = EdgeInsets(top: 16, leading: 16, bottom: 16, trailing: 16),
         onTapExit: (() -> Void)? = nil,
-        @MapViewContentBuilder makeMapContent: () -> [StyleLayerDefinition] = { [] }
+        @MapViewContentBuilder makeMapContent: () -> [StyleLayerDefinition] = { [] },
+        @ViewBuilder currentRoadNameViewBuilder: @escaping (String?) -> AnyView = { name in
+            AnyView(CurrentRoadNameView(currentRoadName: name, theme: DefaultRoadNameViewTheme()))
+        }
```

Comment:
Use view modifier pattern?

Reply:
Yeah; that might actually be the right move here...